Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -63,10 +62,11 @@ extension NavigationMapView {
          Find the count of remaining points in the current step.
          */
         let lineString = currentStepProgress.step.shape ?? LineString([])
-        if currentStepProgress.distanceTraveled < 0 {
-            allRemainingPoints += currentLegSteps[currentLegProgress.stepIndex].count
-        } else if let slicedLineString = lineString.trimmed(from: currentStepProgress.distanceTraveled, to: currentStepProgress.step.distance) {
-            allRemainingPoints += slicedLineString.coordinates.dropLast().count
+        if currentStepProgress.distanceTraveled <= 0 {
```

Comment:
Since we're not using `LineString.trimmed(from:to:)` anymore does it make sense to remove Turf from explicit dependencies?

Reply:
Sure, I'll remove the Turf from explicit dependencies.